lib/logger.js

```diff
@@ -28,8 +30,8 @@ if (config) {
     enabled: config.logging.enabled
   }
 
-  // create the "real" logger
-  module.exports = new Logger(options)
+  // configure logger
+  module.exports.configure(options)
```

Comment:
now that we aren't redefining the export you can remove the `config.setLogger` below.

Reply:
the one use case we intend to support which does not appear to have a test is we started out as `stream: process.stdout`.  This did 1 good thing and obviously 1 bad thing which you're addressing.  But what needs to be handled is if when we get the config it failed to parse we need to set the stream to process.stdout. I think that's as simple as adding 2 items here https://github.com/newrelic/node-newrelic/blob/main/lib/config/index.js#L1715
```_configInstance = Object.assign(defaultConfig(), { agent_enabled: false, logging: { enabled: true, filepath: process.stdout }})```

Reply:
Makes total sense.  Thanks for catching that.